syntax = "proto3";
import "google/protobuf/empty.proto";
package rg2gs;
option cc_generic_services = true;

message Rg2gTest
{}

service Rg2gService {
  rpc EnterRoom (Rg2gTest) returns (google.protobuf.Empty);
}